sdks/java/io/elasticsearch-tests/elasticsearch-tests-common/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTestCommon.java

```diff
@@ -619,4 +619,25 @@ public void testWriteRetry() throws Throwable {
 
     pipeline.run();
   }
+
+  public void testWriteRetryValidRequest() throws Throwable {
```

Comment:
Nit 1: -public

Nit 2:
This is almost the same as the `testWrite()`. To avoid unnecessary code duplication perhaps this should be refactored so both call a `executeWriteTest(ElasticsearchIO.Write write)`

Reply:
done